packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -1072,10 +1072,7 @@ function intOrFloat(value: number): string {
 }
 
 function enumName(name: string): string {
-  if (name.toUpperCase() === name) {
-    return name;
-  }
-  return applyCasing(name, { casing: CASING }).toUpperCase();
```

Comment:
@joheredi do you have any idea why we need to use upper case for enum member name ?